apps/passport-server/test/devconnect.spec.ts

```diff
@@ -1053,95 +1159,101 @@ describe("devconnect functionality", function () {
    * This test updates an event, runs a pretix sync, then fetches the
    * affected PCD to see if the new event name is reflected there.
    */
-  step("should see event updates reflected in PCDs", async function () {
-    const updatedName = "New name";
+  step(
```

Comment:
I wonder if we can enforce some of this code layout stuff in pre-commit hooks so that our PRs aren't full of each other's editor settings fighting over how code should be laid out.

Reply:
#533